pyre/extra_tests/snippets/attribute_error_lookup_context.py

```
@@ -0,0 +1,42 @@
+
+class Empty:
+    def __getattr__(self, name):
+        raise AttributeError()
+
+
+empty = Empty()
+name = "".join(["mis", "sing"])
+try:
+    getattr(empty, name)
+except AttributeError as exc:
+    assert exc.name is name
```

Ruff (0.16.1)

[warning] Found assertion on exception `exc` in `except` block, use `pytest.raises()` instead

(PT017)